### PR TITLE
Remove unused parameters

### PR DESCRIPTION
**Pull request by cston**

### CODE REVIEW

**Comment by cston** [comment]
@dotnet/roslyn-compiler please review.

File: src/Compilers/CSharp/Portable/Syntax/SyntaxFactory.cs [hunk 1]
```diff
@@ -1047,7 +1047,7 @@ public static XmlTextAttributeSyntax XmlTextAttribute(string name, params Syntax
         /// <param name="textTokens">A list of tokens used for the value of the xml text attribute.</param>
         public static XmlTextAttributeSyntax XmlTextAttribute(string name, SyntaxKind quoteKind, SyntaxTokenList textTokens)
         {
-            return XmlTextAttribute(XmlName(name), SyntaxKind.DoubleQuoteToken, textTokens);
+            return XmlTextAttribute(XmlName(name), quoteKind, textTokens);
```
Comment (gafter) [inline src/Compilers/CSharp/Portable/Syntax/SyntaxFactory.cs]: This looks like a real bug fix?

File: src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerExecutor.cs [hunk 2]
```diff
@@ -127,7 +132,7 @@ internal class AnalyzerExecutor
                 onAnalyzerException: onAnalyzerException,
                 analyzerExceptionFilter: null,
                 analyzerManager: analyzerManager,
-                analyzerExecutionTimeMapOpt: null,
+                analyzerExecutionTimeMapOpt: CreateAnalyzerExecutionTimeMap(logExecutionTime),
```
Comment (gafter) [inline src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerExecutor.cs]: Why the change here?

**Comment by cston** [comment]
@mavasani, @johnhamby please review previously unused `logExecutionTime` parameter in `AnalyzerExecutor.CreateForSupportedDiagnostics`.

File: src/Compilers/Core/Portable/PortableShim.cs [hunk 3]
```diff
@@ -577,7 +577,7 @@ internal static int TransformBlock(object hashAlgorithm, byte[] inputBuffer, int
                     s_lazyTransformBlock = TypeOpt.GetTypeInfo().GetDeclaredMethod(nameof(TransformBlock), new[] { typeof(byte[]), typeof(int), typeof(int), typeof(byte[]), typeof(int) });
                 }
 
-                return (int)s_lazyTransformBlock.Invoke(hashAlgorithm, new object[] { inputBuffer, inputOffset, inputCount, inputBuffer, inputOffset });
+                return (int)s_lazyTransformBlock.Invoke(hashAlgorithm, new object[] { inputBuffer, inputOffset, inputCount, outputBuffer, outputOffset });
```
Comment (gafter) [inline src/Compilers/Core/Portable/PortableShim.cs]: thank you!

**Comment by gafter** [comment]
LGTM, aside from the questions asked herein.

File: src/Compilers/CSharp/Portable/Syntax/SyntaxFactory.cs (hunk 1 above)
Comment (cston) [inline src/Compilers/CSharp/Portable/Syntax/SyntaxFactory.cs]: Yes, I believe this was a bug.

File: src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerExecutor.cs (hunk 2 above)
Comment (cston) [inline src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerExecutor.cs]: `logExecutionTime` was previously unused. @mavasani was that intentional?

Comment (mavasani) [inline src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerExecutor.cs]: I would actually remove the parameter `logExecutionTime` from this method. None of the current callsites pass in an explicit value for this flag, and hence will always be false.

Comment (cston) [inline src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerExecutor.cs]: Removed, thanks.

**Comment by VSadov** [comment]
LGTM

**Comment by sharwell** [comment]
@JoeRobich It sure would be nice to be able to go back through old merged PRs and assign the right milestones :)

**Comment by JoeRobich** [comment]
Wow. You mean like all the way back. =)
